Dockerfile

```diff
@@ -16,7 +16,10 @@ COPY --chown=node:node package-lock.json .
 RUN npm ci --legacy-peer-deps
 RUN npm cache clean --force
 
-COPY --chown=node:node . .
+COPY --chown=node:node src src
+COPY --chown=node:node nest-cli.json nest-cli.json
```

Comment:
We're ignoring this file in the `.dockerignore` file currently

Reply:
good spot! - updated now